### PR TITLE
Chore/mui v5

### LINKED ISSUE #33: Upgrade to Material UI V5

### PR DESCRIPTION
**Pull request by storiesOfRen**
migrate from mui v4 to v5

### CODE REVIEW

File: components/Image.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { ReactElement } from 'react';
 import NextImage from 'next/image';
-import { Avatar, createStyles, makeStyles, Theme } from '@material-ui/core';
+import { Avatar, Theme } from '@mui/material';
+import createStyles from '@mui/styles/createStyles';
```
Comment (heystevegray) [inline components/Image.tsx:4]: Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```

File: components/layout.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import { makeStyles, createStyles, Typography } from '@material-ui/core';
+import { Typography } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
```
Comment (heystevegray) [inline components/layout.tsx:2]: Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```

File: components/list/ListItem.tsx [hunk 3]
```diff
@@ -1,15 +1,7 @@
...
-    Avatar,
-    ListItemText,
-    Grid,
-    Typography,
-    makeStyles,
-    createStyles,
-    Theme,
-} from '@material-ui/core';
+import { ListItem as MUIListItem, ListItemAvatar, Avatar, ListItemText, Grid, Typography, Theme } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
```
Comment (heystevegray) [inline components/list/ListItem.tsx:3]: Could we combine these from `@mui/styles`?
```suggestion
import { makeStyles, createStyles  } from '@mui/styles;
```

File: lib/theme.ts [hunk 4]
```diff
@@ -10,7 +9,7 @@ declare module '@material-ui/core/styles/createTypography' {
     }
 }
 
-const THEME: ThemeOptions = {
+const THEME: DeprecatedThemeOptions = {
```
Comment (heystevegray) [inline lib/theme.ts:12]: Same as the other template, is it possible to make this the new `ThemeOptions` without too much headeache? Thank you.

File: tests/test-utils.tsx [hunk 5]
```diff
@@ -4,9 +4,14 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { RouterContext } from 'next/dist/shared/lib/router-context';
 import { NextRouter } from 'next/router';
-import { ThemeProvider } from '@material-ui/core';
+import { ThemeProvider, Theme, StyledEngineProvider } from '@mui/material';
 import theme from '../lib/theme';
 
+declare module '@mui/styles/defaultTheme' {
```
Comment (heystevegray) [inline tests/test-utils.tsx:10]: Also similar, we should only need to declare this once right? We have this in the `pages/_app.tsx` file.

**Review by keonik — APPROVED** [review]

**Comment by keonik** [comment]
@all-contributors please add @storiesOfRen for code

**Pull request merged**